site/content/docs/5.2/getting-started/webpack.md

```diff
@@ -1,112 +1,220 @@
 ---
 layout: docs
-title: Webpack and bundlers
-description: Learn how to include Bootstrap in your project using Webpack or other bundlers.
+title: "Bootstrap & Webpack"
+description: The official guide for how to include and bundle Bootstrap's CSS and JavaScript in your project using Webpack.
 group: getting-started
 toc: true
 ---
 
-## Installing Bootstrap
+{{< callout >}}
+**Want to skip to the end?** Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/webpack). You can also [open the example in Stackblitz](https://stackblitz.com/github/twbs/examples/tree/main/webpack?file=index.html) for live editing.
+{{< /callout >}}
 
-[Install bootstrap]({{< docsref "/getting-started/download#npm" >}}) as a Node.js module using npm.
+## Setup
 
-## Importing JavaScript
+We're building a Webpack project with Bootstrap from scratch, so there are some prerequisites and up front steps before we can really get started. This guide requires you to have Node.js installed and some familiarity with the terminal.
 
-Import [Bootstrap's JavaScript]({{< docsref "/getting-started/javascript" >}}) by adding this line to your app's entry point (usually `index.js` or `app.js`):
+<br>
 
-<!-- eslint-skip -->
-```js
-import 'bootstrap';
+1. **Create a project folder and setup npm.** We'll create the `my-project` folder and initalize npm with the `-y` argument to avoid it asking us all the interactive questions.
```

Comment:
```suggestion
1. **Create a project folder and setup npm.** We'll create the `my-project` folder and initialize npm with the `-y` argument to avoid it asking us all the interactive questions.
```